app/models/search/pseud_indexer.rb

```diff
@@ -18,7 +18,7 @@ def self.mapping
           },
           byline: {
             type: "text",
-            analyzer: "simple"
+            analyzer: "standard"
```

Comment:
For [reference](https://www.elastic.co/guide/en/elasticsearch/reference/6.0/analysis-analyzers.html):

> The standard analyzer divides text into terms on word boundaries, as defined by the Unicode Text Segmentation algorithm. It removes most punctuation, lowercases terms, and supports removing stop words. 

and

> The simple analyzer divides text into terms whenever it encounters a character which is not a letter. It lowercases all terms.